app/models/tag.rb

```diff
@@ -40,6 +106,18 @@ def delete_and_update!
         tag_name:       name
       }
     )
-    destroy
+
+    # Create the delete activity.
+    repository.create_activity(
+      :delete,
+      owner:      actor,
+      recipient:  self,
```

Comment:
Not really, because the final render will pick up the stuff stored in the parameters. The thing here is that we rely on the `parameters` when stuff is deleted.